com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -570,8 +570,8 @@ void DoSubpixelMorphologicalAntialiasing(ref CameraData cameraData, CommandBuffe
                 stencil = m_Depth.Identifier();
                 tempDepthBits = 0;
             }
-            cmd.GetTemporaryRT(ShaderConstants._EdgeTexture, GetCompatibleDescriptor(m_Descriptor.width, m_Descriptor.height, m_SMAAEdgeFormat, tempDepthBits), FilterMode.Point);
-            cmd.GetTemporaryRT(ShaderConstants._BlendTexture, GetCompatibleDescriptor(m_Descriptor.width, m_Descriptor.height, GraphicsFormat.R8G8B8A8_UNorm), FilterMode.Point);
+            cmd.GetTemporaryRT(ShaderConstants._EdgeTexture, GetCompatibleDescriptor(m_Descriptor.width, m_Descriptor.height, m_SMAAEdgeFormat, tempDepthBits), FilterMode.Bilinear);
+            cmd.GetTemporaryRT(ShaderConstants._BlendTexture, GetCompatibleDescriptor(m_Descriptor.width, m_Descriptor.height, GraphicsFormat.R8G8B8A8_UNorm), FilterMode.Bilinear);
```

Comment:
Why does it fail specifically in GL? Not sure if filtering was wrong or used point for optimization. This will possibly cause a performance regression in all platforms?

Reply:
GL does not support sampler states or inline sampler states on graphics shaders, so the filtering needs to be set in the URP code. That's why other APIs were not failing. And about the performance regression, it shouldn't have an impact because the textures we're being sampled linear before too, but it was being done in the shader.